content/views.py

```diff
@@ -117,3 +117,116 @@ def get(self, request, content_id):
+            content     = Content.objects.get(id=content_id)
+            people_list = content.contentpeople_set.all()
+
+            result = [
+                {
+                    'people_id'        : people.people.id,
+                    'people_name'      : people.people.name,
+                    'role'             : people.role_name,
+                    'people_image_url' : people.people.profile_image_url,
+                } for people in people_list
+            ]
+            return JsonResponse({'MESSAGE' : 'SUCCESS', 'RESULT' : result}, status=200)
+        except Content.DoesNotExist:
+            return JsonResponse({'MESSAGE' : 'INVALID_CONTENT_ID'}, status=400)
+
+
+class WatchaContent(View):
+    def get(self, request):
+        service = ContentService.objects.get(name='watcha')
+        watcha_contents = ContentAvailableService.objects.filter(content_service_id=service.id)
+        watcha_content_list = [
+            {
+                'id'             : watcha_content.content.id,
+                'title_korean'   : watcha_content.content.title_korean,
+                'title_original' : watcha_content.content.movieoverview_set.get().title_original,
+                'category'       : watcha_content.content.category.name,
+                'main_image_url' : watcha_content.content.main_image_url,
+            } for watcha_content in watcha_contents
+        ]
+
+        results = [
+            {
+                'service_id'   : service.id,
+                'service_name' : service.name,
+                'contents'     : watcha_content_list,
+            }
+        ]
+        return JsonResponse({'MESSAGE' : 'SUCCESS', 'RESULT' : results}, status=200)
+
+
+class NetflixContent(View):
+    def get(self, request):
+        service              = ContentService.objects.get(name='netflix')
+        netflix_contents     = ContentAvailableService.objects.filter(content_service_id=service.id)
+        netflix_content_list = [
+            {
+                'id'             : netflix_content.content.id,
+                'title_korean'   : netflix_content.content.title_korean,
+                'title_original' : netflix_content.content.movieoverview_set.get().title_original,
+                'category'       : netflix_content.content.category.name,
+                'main_image_url' : netflix_content.content.main_image_url,
+            } for netflix_content in netflix_contents
+        ]
+
+        results = [
+            {
+                'service_id'   : service.id,
+                'service_name' : service.name,
+                'contents'     : netflix_content_list,
+            }
+        ]
+        return JsonResponse({'MESSAGE' : 'SUCCESS', 'RESULT' : results}, status=200)
+
+
+class ContentGallery(View):
+    def get(self, request, content_id):
+        try:
+            content            = Content.objects.get(id=content_id)
+            gallery_photos     = ContentPhoto.objects.filter(content_id=content_id)
```

Comment:
`content`로 접근해서 모든 데이터를 조회해 올 수 있습니다.
```suggestion
            content            = Content.objects.get(id=content_id)
```

Reply:
반영했습니다.